devops/infrastructure-as-code/aws/cli-prefect2-ecs-agent/Readme.md

```diff
@@ -0,0 +1,40 @@
+# Prefect Agent on ECS Fargate
+
+## Purpose
+This recipe will walk you through the process to create a Prefect Agent using the AWS CLI.
+
+### Notes
+- This guide provides an example for creating a role which allows S3 read access, it is likely that your agent will require [additional roles](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create.html), creation of which is outside the scope of this guide.
+
+## Prerequisites
+* [awscli]("https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html")
+* [prefect](https://docs.prefect.io/getting-started/installation/)
+
+
+## Steps
+1. Create a Service Account in Prefect Cloud
+2. Copy the JSON below into a file called `prefect-agent-td.json`
+	1. Edit the file and fill in values for:
+		1. `WORK_QUEUE_ID`
+		2. `PREFECT_API_KEY`
+		3. `PREFECT_API_URL`
+3. Optional: Create a role with S3 read permissions to attach to the agent:
+   1. `aws iam create-role --role-name PrefectECSRole --assume-role-policy-document file://trust-policy.json`
+   2. `aws iam attach-role-policy --policy-arn arn:aws:iam::aws:policy/AmazonS3ReadOnlyAccess --role-name PrefectECSRole`
+4. Use AWS CLI to [register your task definition](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definitions.html)
+    ```
+    aws ecs register-task-definition --cli-input-json file://<full_path_to_task_definition_file>/prefect-agent-td.json
+    ```
+5. Create a service from your task definition, taking care to fill in any network-configuration:
```

Comment:
Are there additional details we could provide for someone who is unfamiliar with creating a service from the task definition?